scripts/install-chef-suse.sh

```diff
@@ -277,6 +286,19 @@ ensure_service_running () {
     fi
 }
 
+cleanup_steps () {
+    for i in pre_sanity_checks \
+         run_services \
+         initial_chef_client \
+         barclamp_install \
+         bootstrap_crowbar_setup \
+         apply_crowbar_config \
+         transition_crowbar \
+         chef_client_daemon \
+         post_sanity_checks; do
+         rm -f /var/lib/crowbar/${i}
```

Comment:
Oh, you actually create them there already ;-) I guess I gave that feedback earlier, but the update is missing here.